docs/baseband/gsb/index.rst

```diff
@@ -9,6 +9,224 @@ the `Giant Metrewave Radio Telescope (GMRT) <http://www.gmrt.ncra.tifr.res.in/>`
 Specifications, design notes and operating procedures for the software
 backend are found 
 `here <http://gmrt.ncra.tifr.res.in/gmrt_hpage/sub_system/gmrt_gsb/index.htm>`_.
+Its design document is found `on the ArXiv <https://arxiv.org/abs/0910.1517>`_.
+
+.. _gsb_file_structure:
+
+File Structure
+==============
+
+A GSB dataset consists of an ASCII file with a sequence of
+:term:`headers<header>`, and one or more accompanying binary data files.  Each
+line in the header and its corresponding data comprise a :term:`data frame`,
+though these do not have explicit divisions in the data files.
+
+Baseband currently supports two forms of GSB data: **rawdump**, for storing
+real-valued raw voltage timestreams, and **phased**, for storing complex
+pre-channelized data from the GMRT in phased array mode.
+
+Rawdump, (`specification document
+<http://gmrt.ncra.tifr.res.in/gmrt_hpage/sub_system/gmrt_gsb/GSB_rawdump_data_format_v2.pdf>`_)
+has a single data file and accompanying header representing the voltage stream
+from one polarization of a single dish.  The header is just the computer-based
+timestamp (accurate to :math:`1\,\mathrm{\mu s}`)::
+
+    YYYY MM DD HH MM SS.SSSSSS
+
+Rawdump is recorded at a rate of 33.3333... megasamples per second (Msps). 
+Each sample is 4 bits in size, and two samples are grouped into 8-bit words
+such that the oldest sample occupies the least significant bit.  Each data
+frame consists of **4 megabytes** worth, or :math:`2^{23}`, samples; as such,
+the timespan of one frame is exactly **251.658240 ms**.
+
+Data in phased format is spread over multiple binary files and one
+accompanying header file.  The right and left circular polarization data are
+separated into two separate sets of files.  Each polarization is further
+subdivided into multiple files with staggered timestreams such that each file
+contains a portion of every frame.  For example, a typical GSB observation may
+contain four files, two for each polarization.  For each polarization, the
+first half of the payload data of each frame is stored in one file, and the
+second half is stored in the other.
+
+The "raw", or pre-channelized, :term:`sample rate` is either 33.3333... Msps at
```

Comment:
Here, definitely there is an 8MHz bandwidth setting.

Reply:
The [GSB design paper](https://arxiv.org/pdf/0910.1517.pdf) Sec. 2.3.1, and the diagram directly above it, makes it sound like only 16 and 32 MHz bandwidth modes exist.  If 8.333... MHz bandwith definitely exists, I'll write that 16 MHz / 8 bps and 32 MHz / 4 bps are the default modes.